dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -62,10 +69,13 @@
     @GET
     @Path("/tools/{organization}")
     @UnitOfWork(readOnly = true)
-    @Produces({ "application/json", "text/plain" })
-    @ApiOperation(value = "List tools of an organization", notes = "This endpoint returns tools of an organization. ", response = ToolV1.class, responseContainer = "List")
+    @Produces({ MediaType.APPLICATION_JSON })
```

Comment:
You dropped `text/plain` -- is that on purpose?

Reply:
yup